FordLabs/PeopleMover
# Add hover state to sort by dropdown

## Issue
Connects #207 

## What was done
- [x] Added hover state to sort by dropdown

## How to test
1. Click on the "sort by" dropdown
2. Hover over an option
3. Ensure the hover state matches the account dropdown
